File: Mobile/media_tracker_test/lib/models/tmdb/tmdb_movie.dart
class TMDBMovie {
  final int id;
  final String title;
  final String overview;

  TMDBMovie({required this.id, required this.title, required this.overview});

  factory TMDBMovie.fromJson(Map<String, dynamic> json) {
    return TMDBMovie(
      id: json['id'],
      title: json['title'],
      overview: json['overview'],
    );
  }
}

File: Desktop/media-tracker-desktop/media-tracker-desktop/Models/TMDB/TMDB_Movie.cs
using Newtonsoft.Json;
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace media_tracker_desktop.Models.TMDB
{
    public class TMDB_Movie
    {
        [JsonProperty("id")]
        public int ID { get; }

        [JsonProperty("title")]
        public string Title { get; } = string.Empty;

        [JsonProperty("overview")]
        public string Overview { get; } = string.Empty;

        [JsonConstructor]
        public TMDB_Movie(int id, string title, string overview)
        {
            this.ID = id;
            this.Title = title;
            this.Overview = overview;
        }
    }
}
